client/packages/invoices/src/OutboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx

```diff
@@ -19,26 +19,24 @@ const RelatedDocumentsSectionComponent = () => {
   const { localisedDate: d } = useFormatDateTime();
   const { requisition } = useOutbound.document.fields('requisition');
 
-  let tooltip = '';
-  if (requisition) {
-    const { user, createdDatetime } = requisition;
-    tooltip = t('messages.customer-requisition-created-on', {
-      date: d(new Date(createdDatetime)),
+  const getTooltip = (createdDatetime: string, username?: string) => {
+    let tooltip = t('messages.customer-requisition-created-on', {
+      date: d(createdDatetime),
```

Comment:
if `requisition` is undefined (which it can't be due to the conditional) then so is the `createdDatetime` and then DateUtils throws an error. We will also have a problem if the date format is problematic and can't be parsed.

I wonder about updating `DateUtils` like this: https://github.com/openmsupply/open-msupply/compare/726-invoice-issues-%231...726-invoice-issues-%231-dates

Since I've had this problem already elsewhere 😉